### ISSUE TITLE
[3.0.0.0] Theme can't be edited because of "theme_default" reference

### ISSUE THREAD

**Issue by qphoria** [opencart/opencart#5598]
In admin/controller/extension/theme/default.php

The validation checks for user permissions to
`if (!$this->user->hasPermission('modify', 'extension/theme/theme_default')) {`
The correct path should be
`if (!$this->user->hasPermission('modify', 'extension/theme/default')) {`

**Comment by harnishdesign** [comment]
I have install fresh latest OpenCart Version 3.0.1.1. but, if try to change any settings on default theme settings, does not save theme settings in admin side option.

I think, We need to change `default` to `theme_default` in following code.
`admin\controller\extension\theme\default.php`

```
if (isset($this->request->get['store_id']) && ($this->request->server['REQUEST_METHOD'] != 'POST')) {
			$setting_info = $this->model_setting_setting->getSetting('default', $this->request->get['store_id']);
		}
```
to

```
if (isset($this->request->get['store_id']) && ($this->request->server['REQUEST_METHOD'] != 'POST')) {
			$setting_info = $this->model_setting_setting->getSetting('theme_default', $this->request->get['store_id']);
		}
```
working for me after this changes. please let us know. if, I am wrong.

**Comment by danielkerr** [comment]
download from the master please

there is a fix already added.

**Comment by karkawy** [comment]
From where i can get this fix? My op 3.0.2.0 ..cant enable custom theme ( the save button not work)
